packages/settings-library/src/settings_library/utils_service.py

```diff
@@ -0,0 +1,53 @@
+""" Helpers to build settings for services with http API
+
+
+"""
+
+
+from pydantic.networks import HttpUrl
+
+from .basic_types import PortInt
+
+DEFAULT_AIOHTTP_PORT: PortInt = 8000
+DEFAULT_FASTAPI_PORT: PortInt = 8080
+
+
+class MixinServiceSettings:
+    """
+    Subclass should define fields:
+
+    class MyServiceSettings(BaseCustomSettings, MixinServiceSettings):
+        {prefix}_HOST
```

Comment:
you might be missing the scheme maybe... as example 90% of the time it will be HTTP.. but for instance with dask we also have TCP or TLS... Not for now... just as a reminder
actually file:// is also valid... or ftp:// or smb://